treeherder/services/pulse/consumers.py

```diff
@@ -92,7 +93,7 @@ def bind_to(self, exchange, routing_key):
                 exchange=exchange,
                 routing_key=routing_key,
                 durable=True,
-                auto_delete=False,
+                auto_delete=True,
```

Comment:
This cannot always be `True` since in production we don't want the queues to be deleted.
Sometimes infrastructure can disconnect and we don't want messages in the queues to be deleted.

We want production to be False and we want Heroku Review apps to be True.
I won't give you how to solve you but for you to propose how to solve it.
The env variable is the cluse. Make the change in `treeherder.config.settings`.